dts/bindings/sensor/st,lis2dh.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2018 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: STMicroelectronics MEMS sensors LIS2DH
+version: 0.1
+
+description: >
+    This binding gives a base representation of LIS2DH 3-axis accelerometer
+
+inherits:
+    !include i2c-device.yaml
+
+properties:
+    compatible:
+      constraint: "st,lis2dh"
```

Comment:
@galak, should this be "st,lis2dh-i2c"?

Reply:
@MaureenHelm 
> should this be "st,lis2dh-i2c"

My opinion is that it does make sense indeed, but if this is a general rule that has to be followed there are other sensors (e.g. lsm6dsl) that need to be changed too.